packages/template-retail-react-app/app/pages/account/profile.jsx

```diff
@@ -60,7 +62,7 @@ const Skeleton = forwardRef(({children, height, width, ...rest}, ref) => {
 
 Skeleton.displayName = 'Skeleton'
 
-const ProfileCard = () => {
+const ProfileCard = ({isEcomAccount = false}) => {
```

Comment:
There might be other use cases for wanting to disable the password reset. Might be wise to make this a little more generic by making this literal.. like "canResetPassword" or "allowPasswordChange". This way the component will be more flexible.